cherrypicks.go

```diff
@@ -100,7 +100,11 @@ func suggestCherryPicks(log *logrus.Entry, pr *github.PullRequestEvent, githubCl
 		if err != nil {
 			return err
 		} else if releaseBranch != "" {
-			releaseBranches = append(releaseBranches, releaseBranch+" (release "+version+")")
+			if isBottable(pr.GetRepo().GetName(), conf, pr.GetPullRequest(), releaseBranch) {
```

Comment:
I personally like the name, `isBottable`, but it really is not a very descriptive name, it should at least indicate that this is about cherry-picking and not any other "bot" actionable thing. Suggestion: `isCleanCherryPick`, or `isCherryPickBottable`.